Ginger/Ginger/UserControlsLib/UCDataMapping.xaml.cs

```diff
@@ -0,0 +1,553 @@
+#region License
+/*
+Copyright © 2014-2020 European Support Limited
+
+Licensed under the Apache License, Version 2.0 (the "License")
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at 
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+#endregion
+
+using amdocs.ginger.GingerCoreNET;
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.Repository;
+using Ginger.Actions;
+using GingerCore.DataSource;
+using GingerCore.GeneralLib;
+using GingerCore.Variables;
+using System;
+using System.ComponentModel;
+using System.Linq;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Media;
+
+namespace Ginger.UserControlsLib
+{
+    public partial class UCDataMapping : UserControl, INotifyPropertyChanged
+    {
+        public static DependencyProperty MappedTypeProperty =
+        DependencyProperty.Register("MappedType", typeof(string), typeof(UCDataMapping), new PropertyMetadata(OnMappedTypePropertyChanged));
+
+        public static DependencyProperty MappedValueProperty =
+        DependencyProperty.Register("MappedValue", typeof(string), typeof(UCDataMapping), new PropertyMetadata(OnMappedValuePropertyChanged));
+
+        public static DependencyProperty EnableDataMappingProperty =
+        DependencyProperty.Register("EnableDataMapping", typeof(bool), typeof(UCDataMapping), new PropertyMetadata(OnEnableDataMappingPropertyChanged));
+
+        public static DependencyProperty VariabelsSourceProperty =
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'VariabelsSourceProperty' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5538205)